specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2022-03-02-preview/managedClusters.json

```diff
@@ -6298,6 +6307,14 @@
       },
       "description": "Only apply to AAD clusters, specifies the format of returned kubeconfig. Format 'azure' will return azure auth-provider kubeconfig; format 'exec' will return exec format kubeconfig, which requires kubelogin binary in the path.",
       "x-ms-parameter-location": "method"
+    },
+    "IgnorePodDisruptionBudgetParameter": {
+      "name": "ignore-pod-disruption-budget",
+      "in": "query",
+      "required": false,
+      "type": "string",
```

Comment:
Can this be a bool instead? Strings that can only be "true" or "false" are very awkward to use in SDKs, etc.

Reply:
Changed to boolean type.